### PR TITLE
LD4RAIL-1069 migration ontology 3.1.0 (11-11-2024)

### PR DESCRIPTION
**Pull request by ednaru**
Fixes:
- Added domain CommonCharacteristicsSubset to infrastructureManager property
- Added Plafform, Siding to domain of property tenClassificaion
- Added functional axiom to Siding id
- Deprecated era:track and era:siding properties (replaced by era:hasPart)
- Deprecated imCode, replaced by organisationCode

### CODE REVIEW

File: ontology.ttl [hunk 1]
```diff
@@ -11820,16 +11830,44 @@ foaf:Person rdf:type owl:Class ;
...
 
 
+[ foaf:name "Designated RINF Topical Working Groups"@en
+] .
+
+[ foaf:name "Oscar Corcho"
+ ] .
+
+[ foaf:name "Edna Ruckhaus"
+ ] .
```
Comment (gatemezing) [inline ontology.ttl:11840]: why do you have this duplicate section for authors and contributors? lines 11833 - 11861? Could you delete those lines if repeated?

**Review by gatemezing — CHANGES_REQUESTED** [review]
Please, double check the duplicated section for contributors and authors. Is it Protége? Thank you!

File: ontology.ttl (hunk 1 above)
Comment (ednaru) [reply ontology.ttl]: Protégé duplicates these lines.

**Review by gatemezing — APPROVED** [review]
good to go!

**Pull request merged**